GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -85,20 +88,31 @@ public void EnsureAuthenticated()
                     uri = new Uri(Url);
                 }
 
-                // TODO: Use TfsTeamProjectCollection constructor that takes a TfsClientCredentials object
+#if VS2010
```

Comment:
If it's just compiler warnings being squashed, how about using [`#pragma warning disable X`](https://msdn.microsoft.com/en-us/library/441722ys.aspx)?